modules/electrophysiology_browser/README.md

```diff
@@ -16,20 +17,24 @@ been correctly inserted into LORIS.
 
 The Electrophysiology Browser displays electrophysiology datasets that have been
 inserted into LORIS from a BIDS-format collection. Derived or processed electrophysiology
-datasets can also be accessed via this module.
+datasets can also be accessed and edited via this module.
 
 ## Permissions
 
-The Electrophysiology Browser uses the following permissions. Any of them is 
-sufficient to provide access to view data in the module.
+The Electrophysiology Browser uses the following permissions. Either of the first two is 
+sufficient to provide access to view data in the module. The third permission provides editing
+permissions for derivative datasets.
 
 electrophysiology_browser_view_allsites
   - This permission gives the user access to all electrophysiology datasets present in the database.
   
 electrophysiology_browser_view_site
   - This permission gives the user access to electrophysiology datasets from their own site(s) only.
+  
+electrophysiology_browser_edit_annotations
+  - This permission allows the user to add, edit, and delete annotations for raw or derived datasets
 
 ## Download
 
-You can download all the files related to a recording (channel information,
-electrode information, task event information, the actual recording...).
+You can download all the files related to a recording and its derivatives (channel information,
+electrode information, task event information, the actual recording, the annotations and their metadata...).
```

Comment:
```suggestion
You can download all the files related to a recording (channel information,
electrode information, task event information, the actual recording) -- as well as its derivatives (annotations and their metadata).
```
updated this for clarity.  also no ellipses in documentation if you can help it...